app/decorators/member_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class MemberDecorator < Draper::Decorator
+  delegate_all
+
+  def show_email
+    User.find(object.user_id).email
```

Comment:
if object has `user_id` so you have `belongs_to` association.... so you can use it 
` object.user.email` also think about validation on presence for user because 
in case when member hasn't assigned user you will get exception here.